source/releasenotes/2024-04-11-wordpress-composer-managed-update.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Updates to WordPress (Composer Managed) Upstream
+published_date: "2024-04-11"
+categories: [wordpress, action-required]
+---
+
+Today we are pushing out an update to the [WordPress (Composer Managed)](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) upstream to resolve a particular log message that started appearing after our [WP-CLI wrapper update](/release-notes/2024/02/wp-cli-changes) in February.
```

Comment:
"An update is now available for the WordPress Composer Managed upstream to resolve ..."